app/api/discount_codes.py

```diff
@@ -133,14 +133,13 @@ def query(self, view_kwargs):
         # user can only access his/her discount codes.
         if view_kwargs.get('user_id'):
             if has_access('is_user_itself', user_id=view_kwargs['user_id']):
-                user = safe_query(self, User, 'id', view_kwargs['user_id'], 'user_id')
+                user = safe_query(User, 'id', view_kwargs['user_id'], 'user_id')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'parameter_name' in function call](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5526478)